src/plugins/intel_gpu/src/graph/pass_manager.cpp

```diff
@@ -49,9 +49,13 @@ void pass_manager::run(program& p, base_pass& pass) {
     using Time = std::chrono::high_resolution_clock;
 
     GPU_DEBUG_LOG << "Run pass " << pass.get_name() << std::endl;
+    GPU_DEBUG_GET_INSTANCE(debug_config);
+    GPU_DEBUG_CODE(cldnn::instrumentation::mem_usage_logger mem_logger{pass.get_name(), false});
+    GPU_DEBUG_CODE(if(debug_config->verbose >= 2) mem_logger.start_logging());
```

Comment:
Why do you limit scope of memory tracker here?
Can't this part be replaced with `GPU_DEBUG_DEFINE_MEM_LOGGER(pass.get_name());` ?

Reply:
It was for more accurate measurement, but in reality it has little effect. Replaced with generic macro